pennylane/devices/default_tensor.py

```diff
@@ -151,28 +181,38 @@ class DefaultTensor(Device):
     This device is designed to simulate large-scale quantum circuits using tensor networks. For small circuits, other devices like ``default.qubit`` may be more suitable.
 
     The backend uses the ``quimb`` library to perform the tensor network operations, and different methods can be used to simulate the quantum circuit.
-    Currently, only the Matrix Product State (MPS) method is supported, based on ``quimb``'s ``CircuitMPS`` class.
+    Currently, the supported methods are Matrix Product State (MPS), based on the ``quimb``'s ``CircuitMPS`` class, and Exact Tensor Network (TN), based on the ``quimb``'s ``Circuit`` class.
```

Comment:
I think "currently" is implied, so I would remove it (here and under) for conciseness. 
```suggestion
    The supported methods are Matrix Product State (MPS), based on the ``quimb``'s ``CircuitMPS`` class, and Exact Tensor Network (TN), based on the ``quimb``'s ``Circuit`` class.
```